installer/manifests/keptn/charts/control-plane/values.yaml

```diff
@@ -75,6 +75,7 @@ apiGatewayNginx:
   nodeSelector: {}
   gracePeriod: 60
   preStopHookTime: 5
+  clientMaxBodySize: "1m"
```

Comment:
I would suggest raising this either to a higher value, or `0`,w hich according to the [docs](http://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size) means that there are no checks for this.
@bacherfl @mowies any concerns regarding that?